pyfr/integrators/dual/pseudo/pseudocontrollers.py

```diff
@@ -140,8 +145,27 @@ def __init__(self, *args, **kwargs):
                     )
                 )
 
+        self.pseudo_step_multiplied(1.0)
+
         self.backend.commit()
 
+    def adjust_pseudo_step(self, dt):
```

Comment:
Maybe change the name of the name of this, similar to `adjust_step`. I think the name isn't very descriptive of what is actually happening